src/wp-includes/blocks.php

```diff
@@ -710,7 +710,13 @@ function excerpt_remove_blocks( $content ) {
 		'core/verse',
 	);
 
-	$allowed_blocks = array_merge( $allowed_inner_blocks, array( 'core/columns' ) );
+	$allowed_wrapper_blocks = array(
```

Comment:
Would there ever be a need to allow this list to be filtered?

Reply:
Are we allowed to introduce a new filter at this stage in the 5.8 release?
If we can, then yes it would be good to have a filter. We'd be able to use it if and when we add new blocks in Gutenberg, and 3rd-party plugins could also potentially use it if they add their own wrapper blocks.
That being said, adding the filter is not critical to this PR and can be done after v5.8 in a separate ticket if now is not the time to add it.

Reply:
Ah, it was not clear to me that this was necessary for 5.8.

In that case, a separate ticket for a filter in 5.9 would be preferred.